pweave/processors/base.py

```diff
@@ -32,10 +35,10 @@ def run(self):
         if self.documentationmode:
             success = self._getoldresults()
             if success:
-                print("Restoring cached results")
+                logger.warn("Restoring cached results")
```

Comment:
I think it should be "info" level (or lower), as it is the expected behaviour of the code.

Reply:
In isolation. I agree. However, I was trying to find and follow some pattern that @mpastell (see https://github.com/mpastell/Pweave/pull/112/files#r178615937). 

If you would still like me to make these changes to the logging levels, I'm very happy to do so.

Reply:
Do you mean `print("Restoring cached results")`?

What I mean is to change every `print(...)` to `logger.info(...)` and every `sys.stderr.write(...)` to `logger.warn(...)`. Sorry in advance if I were inconsistent anywhere.

Reply:
Great, I'll do that!

Reply:
@fgregg I need to amend my previous comment. I forgot https://github.com/mpastell/Pweave/pull/112#discussion_r241118102 which is an exception (from the previous comment).